### ISSUE TITLE
Redefine "F" bands range

### ISSUE THREAD

**Issue by truefriend-cz** [armel/uv-k5-firmware-custom-feat-F4HWN#67]
Hi. How i can change "F" band ranger in VFO mode?

**Comment by armel** [comment]
Hi. Use case ?

**Comment by truefriend-cz** [comment]
In our state, we have different service frequency ranges than the ones that are set as defaults in the firmware. Or I would like to define my own ranges... for example F1 = Trucks (26.8KHz - 28KHz), etc...

**Comment by armel** [comment]
All right. Well, this firmware doesn't offer anything to modify them, I'm sorry. But you can do a Scan Range using F+5 in VFO mode. Take a look at this [video](https://www.youtube.com/watch?v=NYsQ3UtbPKs&ab_channel=FredintheShed) if need.

Armel F4HWN.

**Comment by truefriend-cz** [comment]
> All right. Well, this firmware doesn't offer anything to modify them, I'm sorry. But you can do a Scan Range using F+5 in VFO mode. Take a look at this [video](https://www.youtube.com/watch?v=NYsQ3UtbPKs&ab_channel=FredintheShed) if need.
> 
> Armel F4HWN.

Yes. I know this function. It's just that adding it under these "presets" using "F1, F2, ..., F7" would be more appropriate. Maybe sometime in the future.

**Comment by truefriend-cz** [comment]
CHIRP have problem with your firmware.... 
![image](https://github.com/armel/uv-k5-firmware-custom-feat-F4HWN/assets/7845691/e1c8caf7-5f31-46f1-bf60-3703e033e808)

**Comment by truefriend-cz** [comment]
Do you can add this parameter for modification to CHIRP?

**Comment by armel** [comment]
No, as I said, I don't know if it's possible. But if it's possible, this is a job for Chirp. I'll take a look if I have time.  
But as I said too, it's really a specific need. Even if I understand your need, My firmware tries to take into account very generic requests that are useful to everyone.  I can't say yes to everything and everyone :)

**Comment by f1ixx** [comment]
Hello
if i understand, you must change table : freq_band_table_t frequencyBandTable[]
This table is in <code>frequencie.cpp</code>
Make modification and compile your new firmware.
To have a personal table via "menu" are to big (i'm think)
Or big job by removing the FM ;-)

73's

Boris
